CodingAChat/Scenes/Settings/SettingsCoordinator.swift

```diff
@@ -13,9 +13,11 @@ class SettingsCoordinator: Coordinator {
     var childCoordinators: [Coordinator] = []
     private let viewController: SettingsViewController
     private let presenter: UIViewController
+    private let tracker : Trackable
```

Comment:
colons should always segue identifier with no blank spaces
```suggestion
    private let tracker: Trackable
```